app/models/episode.js

```diff
@@ -13,4 +15,10 @@ export default Base.extend({
   titles: attr('object'),
 
   media: belongsTo('media'),
+
+  computedTitle: computed('number', 'canonicalTitle', function() {
```

Comment:
Since this is only used in a single place, can we move it there so we aren't computing this for every episode we have.